content/forgero-vanilla/src/main/resources/data/forgero/packs/minecraft-material/type/material/bow_limb_material.json

```diff
@@ -0,0 +1,12 @@
+{
+  "parent": "TOOL_MATERIAL",
```

Comment:
Also don't think we need this as the parent, it might cause some wierd cyclic dependencies, so I think we'll leave this one out for now and let it be a standalone type.

Reply:
I tried this and it won't work. I'll have to do a code revision to the type system to make that work properly so I think I'll wait.